### DISCUSSION TITLE
How to upgrade docker/set up Breeze (with corporate proxy)?

### DISCUSSION THREAD

**Discussion by vijaya-lakshmi-venkatraman** [apache/airflow#21713] [Q&A]
Hello,
I am currently setting up Airflow dev env on my RHEL VM with docker v18.03.1-ce
When I try to set up Breeze environment, it says it needs at least V20.10.0 of docker.
I am trying to upgrade docker based on this ref: https://docs.docker.com/engine/install/rhel/#install-using-the-repository 
1) Install from repo gives me "[Errno 14] curl#22 - "HTTP returned error" 
2) Install from package gives me "Cannot add package to transaction. Not a compatible architecture: s390x"
Note: I have tried to supply proxy details but still the issue seems to persist

Has anyone setup Breeze with corporate proxy in place?

**Comment by potiuk** [comment]
This is not really possible. You should have open access to internet to use breeze.
